web/web/api/v1/meter.py

```diff
@@ -10,6 +10,8 @@
 from sqlalchemy.orm.exc import NoResultFound
 from sqlalchemy.orm.exc import MultipleResultsFound
 
+#enables trailing and non-trailing slash routes
+app.url_map.strict_slashes = False
```

Comment:
this should move to the `web/__init__.py` (where `app` gets instantiated) so we don't have to keep setting it in every api endpoint file.

Actually we'll end up moving it to a config file, but that's for future us.

Reply:
Ok! will do thanks